components/measurement/CommonDetails.js

```diff
@@ -35,9 +35,9 @@ const StyledReactJsonContainer = styled.div`
   }
 `
 
-const JsonViewer = ({ src }) => (
+const JsonViewer = ({ src, collapse }) => (
```

Comment:
Use `collapsed` to align with the prop name used by the dependency. It becomes easier to track.